### ISSUE TITLE
Fix `DataFrame.applymap has been deprecated` warning

### ISSUE THREAD

**Issue by Trymzet** [timvink/mkdocs-table-reader-plugin#55]
This is caused by [this line](https://github.com/timvink/mkdocs-table-reader-plugin/blob/781fc82c580f6c0326563a2ff237d389d1a16884/mkdocs_table_reader_plugin/markdown.py#L28C39-L28C88) in `mkdocs_table_reader_plugin/markdown.py`.

**Comment by timvink** [comment]
Thanks for mentioning. I don't want to force users to the lastest version of pandas though..

**Comment by timvink** [comment]
Related #54

**Comment by Trymzet** [comment]
Understandable, but maybe you can add a simple version check?
```python
if pd.__version__.startswith("2"):
     ...
else:
    (current behavior)
```

**Comment by timvink** [comment]
Yeah that's a good solution. Normally I wouldn't worry about these kinds of warnings, but some people run mkdocs using `--strict` mode

**Comment by timvink** [comment]
Just released v2.0.2 with this change!

Thanks for pushing this :)
